prow/plugins/dco/dco.go

```diff
@@ -97,6 +137,32 @@ type commentPruner interface {
 	PruneComments(shouldPrune func(github.IssueComment) bool)
 }
 
+// checkTrustedOrgMember checks are all commits from the trusted org member
+func checkTrustedOrgMember(dco *plugins.Dco, gc gitHubClient, l *logrus.Entry, org, repo string, number int) (bool, error) {
```

Comment:
Here we just check for organization members? Should we check also for collaborators? If yes, should that be default or optional?

Reply:
Look for `prow/plugins/trigger.TrustedUser()`

Reply:
I have updated the function to match what's done by `TrustedUser()`. Let me know if it's okay.